Onboarding System (#844)
italia/eid-wallet-it-docs: docs/it/functionalities.rst

Reviewer: @fmarino-ipzs notiamo che buona parte della revisione riguarda dei termini che in realtà sono definiti nel glossario, di cui avevamo concordato la traduzione rispetto alla versione inglese. È necessario mantenere coerenza tra ciò che è definito nel glossario e come viene ripreso nel testo.

Author: Hai ragione. Per la traduzione in italiano ci siamo affidati a tool automatici che hanno stravolto un po' troppo il testo. Cerchiamo di sistemare asap.

Reviewer: Grazie, siccome la versione italiana l'abbiamo scritta noi (= non è fatta dal traduttore automatico) dovrebbe già essere coerente con la versione inglese. Infatti non mi tornava che nella versione ENG avete proposto 2-3 modifiche, mentre nell'ITA innumerevoli 😅

.. include:: ../common/common_definitions.rst

> Progettazione dell'Esperienza dell'Utente

Reviewer: ```suggestion
Design dell'Esperienza Utente
```
Design è un termine ormai nel vocabolario comune, così come Esperienza Utente è termine definito nel glossario e da letteratura

==========================================

.. include:: design.rst 

Panoramica delle Funzionalità
------------------------------

> Il Sistema IT-Wallet fornisce agli Utenti un modo più semplice, veloce e sicuro per accedere ai servizi. Questo servizio viene erogato attraverso l'uso di una Soluzione Wallet, la cui Esperienza dell'Utente è strutturata in tre fasi principali: pre-uso, uso e post-uso.

Reviewer: ```suggestion
Il Sistema IT-Wallet fornisce agli Utenti un modo più semplice, veloce e sicuro per accedere ai servizi. Questo servizio viene erogato attraverso l'uso di una Soluzione Wallet, la cui Esperienza Utente è strutturata in tre fasi principali: pre-utilizzo, utilizzo e post-utilizzo.
```
Ci sembra più corretta questa forma che poi richiamiamo nei paragrafi successivi.

.. only:: format_html

  .. figure:: ./images/svg/UX-phases-usage.svg

>     :alt: Fasi dell'Esperienza dell'Utente nell'utilizzo del Wallet

Reviewer: ```suggestion
    :alt: Fasi dell'Esperienza Utente nell'utilizzo del Wallet
```

    :width: 100%

>     Fasi dell'Esperienza dell'Utente nell'utilizzo del Wallet

Reviewer: ```suggestion
    Fasi dell'Esperienza Utente nell'utilizzo del Wallet
```

.. only:: format_latex

  .. figure:: ./images/pdf/UX-phases-usage.pdf

>     :alt: Fasi dell'Esperienza dell'Utente nell'utilizzo del Wallet

Reviewer: ```suggestion
    :alt: Fasi dell'Esperienza Utente nell'utilizzo del Wallet
```

    :width: 100%

>     Fasi dell'Esperienza dell'Utente nell'utilizzo del Wallet

Reviewer: ```suggestion
    Fasi dell'Esperienza Utente nell'utilizzo del Wallet
```

> Le sezioni seguenti si concentrano sulle fasi di utilizzo e post-utilizzo. Definiscono i requisiti funzionali che supportano l'Esperienza dell'Utente per le fasi di attivazione, acquisizione, presentazione, gestione e disattivazione, insieme ai requisiti di interazione relativi alla gestione degli errori, alle richieste di assistenza e alla raccolta di feedback.

Reviewer: ```suggestion
Le sezioni seguenti si concentrano sulle fasi di utilizzo e post-utilizzo. Definiscono i requisiti funzionali che supportano l'Esperienza Utente per le fasi di attivazione, ottenimento, presentazione, gestione e disattivazione, insieme ai requisiti di interazione relativi alla gestione degli errori, alle richieste di assistenza e alla raccolta di feedback.
```
non parliamo di acquisizione ma di ottenimento (è l'utente che ottiene) e per coerenza vorremmo mantenere questo termine

Documentazione aggiuntiva e risorse sono fornite nella sezione :ref:`official-resources:Risorse Ufficiali`.  

> Le Official Resources includono raccomandazioni sulle interazioni richieste tra Utente e Istanza del Wallet e le migliori pratiche di progettazione che promuovono coerenza tra diverse Soluzioni Wallet in termini di come le funzionalità vengono accessibili e utilizzate.

Reviewer: ```suggestion
Le Risorse Ufficiali includono raccomandazioni sulle interazioni tra Utente e Istanza del Wallet, e le buone pratiche di progettazione al fine di  promuovere coerenza tra diverse Soluzioni Wallet rispetto alle modalità di fruizione delle funzionalità.
```
Risorse Ufficiali è un termine da glossario

Per garantire un'implementazione corretta e coerente, gli Attori Primari: 

> - DEVONO utilizzare le Official Resources e DEVONO rispettare tutte le specifiche di utilizzo correlate fornite; 

Reviewer: ```suggestion
- DEVONO utilizzare le Risorse Ufficiali e DEVONO rispettare tutte le relative specifiche di utilizzo fornite; 
```

> - POSSONO scegliere tra le configurazioni disponibili fornite. Gli Attori Primari DEVONO garantire l'uso corretto dei componenti atomici, come gli Engagement Button; 

Reviewer: ```suggestion
- POSSONO scegliere quale configurazione implementare, tra quelle rese disponibili, ma DEVONO comunque garantire il corretto utilizzo dei componenti atomici, come l'Engagement Button; 
```

> - DEVONO mantenere aggiornate le risorse utilizzate, in linea con l'ultima versione disponibile delle Official Resources. 

Reviewer: ```suggestion
- DEVONO mantenere aggiornate le risorse utilizzate, in linea con l'ultima versione disponibile nelle Risorse Ufficiali. 
```

Attivazione dell'Istanza del Wallet
------------------------------------

> L'attivazione consente all'Utente di accedere alle funzionalità della Soluzione Wallet per ottenere, presentare e gestire in modo sicuro i propri Attestati Elettronici. Il processo di attivazione coinvolge l'Autenticazione dell'Utente con l'Istanza del Wallet utilizzando la propria identità digitale, che consente la generazione del PID.

Reviewer: ```suggestion
L'attivazione consente all'Utente di accedere alle funzionalità della Soluzione Wallet per ottenere, presentare e gestire in modo sicuro i propri Attestati Elettronici. Il processo di attivazione prevede l'Autenticazione dell'Utente nell'Istanza del Wallet utilizzando la propria identità digitale, che consente la generazione del PID.
```

> Di seguito sono riportati i requisiti dell'Esperienza dell'Utente che il Fornitore di Wallet DEVE garantire tramite la propria Soluzione Wallet:

Reviewer: ```suggestion
Di seguito sono riportati i requisiti dell'Esperienza Utente che il Fornitore di Wallet DEVE garantire tramite la propria Soluzione Wallet:
```

- L'Utente scarica la Soluzione Wallet sul proprio dispositivo per generare la propria Istanza del Wallet;

> - L'Utente imposta un PIN di sblocco per la propria Istanza del Wallet se non è stato precedentemente impostato nell'app. Oltre al PIN, l'Utente può decidere di utilizzare il proprio meccanismo di sblocco utilizzato all'interno del dispositivo e gestito a livello del sistema operativo (ad esempio, autenticazione biometrica) come alternativa al PIN. L'Utente utilizza il metodo di sblocco ogni volta che è richiesta un'autorizzazione per garantire sicurezza e proteggere le proprie informazioni;

Reviewer: ```suggestion
- L'Utente imposta un PIN di sblocco per la propria Istanza del Wallet se non è stato precedentemente impostato nell'app. Oltre al PIN, l'Utente può decidere di utilizzare il metodo di sblocco utilizzato per il dispositivo e gestito a livello di sistema operativo (ad esempio, autenticazione biometrica) come alternativa al PIN. L'Utente utilizza il metodo di sblocco ogni volta che è richiesta un'autorizzazione per garantire sicurezza e proteggere le proprie informazioni;
```

> - L'Utente esamina tutte le informazioni pertinenti riguardanti il processo di attivazione e l'utilizzo del servizio. Inoltre, l'Utente legge qualsiasi policy del Fornitore e del Fornitore di Attestati Elettronici di Dati di Identificazione Personale e/o i termini e le condizioni del servizio. L'Utente dà il proprio consenso per procedere o rifiuta per annullare l'operazione;

Reviewer: ```suggestion
- L'Utente esamina tutte le informazioni pertinenti riguardanti l'attivazione e l'utilizzo del servizio. Inoltre, l'Utente legge eventuali informative del Fornitore di Wallet e del Fornitore di Attestati Elettronici di Dati di Identificazione Personale e/o i termini e le condizioni d'uso del servizio. L'Utente dà il proprio consenso per procedere o rifiuta per annullare l'operazione;
```

A seconda delle specifiche esigenze dell'Utente, della tipologia di Attestato Elettronico di Attributi e delle disponibilità offerte dal Fornitore di Wallet, dal Fornitore di Attestati Elettronici di Attributi e dalla Fonte Autentica, l'ottenimento degli Attestati Elettronici di Attributi può avvenire attraverso due modalità: 

> - **dal Catalogo dell'Istanza del Wallet**: l'Utente esplora l'elenco degli Attestati Elettronici di Attributi forniti dalla Soluzione Wallet, seleziona quello di interesse e avvia il processo di richiesta, concludendo con il rilascio dell'Attestato Elettronico di Attributi nell'Istanza del Wallet. Questo percorso è disponibile per i tipi di credenziale idonei per la discovery pubblica come determinato dalle politiche dell'organismo di supervisione durante il processo di onboarding (vedere :ref:`registry:Catalogo degli Attestati Elettronici`).

Reviewer: ```suggestion
- **dal Catalogo dell'Istanza del Wallet**: l'Utente esplora l'elenco degli Attestati Elettronici di Attributi forniti dalla Soluzione Wallet, seleziona quello di suo interesse e avvia il processo di richiesta, concludendo con l'ottenimento dell'Attestato Elettronico di Attributi nell'Istanza del Wallet. Questa modalità è disponibile per i tipi di credenziale idonei per la discovery pubblica come determinato dalle politiche dell'Organismo di Supervisione durante il processo di onboarding (vedere :ref:`registry:Catalogo degli Attestati Elettronici`).
```
nelle fasi dell'Esperienza Utente non parliamo di rilascio degli EAA ma di ottenimento (user centric)

Reviewer: @fmarino-ipzs

> - **da un Touchpoint della Fonte Autentica** (o del Fornitore di Attestati Elettronici di Attributi se coincide con la Fonte Autentica): l'Utente interagisce con il servizio digitale della Fonte Autentica, consentendogli di ottenere un Attestato Elettronico di Attributi specifico nella propria Istanza del Wallet tramite un Engagement Button. Questo percorso consente l'accesso sia alle credenziali pubblicate nel catalogo che alle credenziali private non idonee per la discovery pubblica (vedere :ref:`registry:Catalogo degli Attestati Elettronici`).

Reviewer: ```suggestion
- **da un Touchpoint della Fonte Autentica** (o del Fornitore di Attestati Elettronici di Attributi qualora coincida con la Fonte Autentica): l'Utente interagisce con il servizio digitale della Fonte Autentica, che gli permette di ottenere un Attestato Elettronico di Attributi specifico nella propria Istanza del Wallet tramite un Engagement Button. Questa modalità consente l'accesso sia alle credenziali pubblicate nel catalogo che alle credenziali non idonee per la discovery pubblica (vedere :ref:`registry:Catalogo degli Attestati Elettronici`).
```
non riteniamo necessario specificare private in quanto le credenziali non idonee per la discovery pubblica possono essere anche pubbliche

Reviewer: @fmarino-ipzs

Reviewer 2: @steps88 you're right, done in 0824575a33cafc17f8f5de37c8bed9a089c877bb
